editor/src/clj/editor/particlefx.clj

```diff
@@ -782,6 +792,7 @@
 (def ^:private default-start-delay (pair pb-default-start-delay pb-default-start-delay-spread))
 
 (g/defnode EmitterNode
+  (inherits core/Scope)
```

Comment:
I refactored the graph a bit to simplify wiring. Previously, all modifiers were connected directly to the ParticleFX node, even when they were children of emitter nodes. I made EmitterNode a scope node and started connecting emitter modifiers here instead. This change simplified the code around connecting modifiers.